src/gui/widgets/multimenu_button.cpp

```diff
@@ -263,7 +121,11 @@ builder_multimenu_button::builder_multimenu_button(const config& cfg)
 	, max_shown_(cfg["maximum_shown"].to_unsigned(1))
 	, options_()
 {
-	for(const auto& option : cfg.child_range("option")) {
+	for(auto option : cfg.child_range("option")) {
+		if(!option.has_attribute("checkbox")) {
+			WRN_GUI_G << "Missing checkbox attribute for multimenu_button menu_item, adding with value=false.";
```

Comment:
Uhhh, what? Is this really worth a warning?

Reply:
It's worth a log message, IMO.  Personally, I don't really like software "correcting" my mistakes, but if it does it better explain why.  What level that message should be I don't know.

Reply:
Even if it's worth a log message, it definitely shouldn't be at warning level.